src/js/utils/ServicesPathUtil.js

```diff
@@ -0,0 +1,20 @@
+const ServicesPathUtil = {
```

Comment:
I don't see any need for this util if you refactor the breadcrumb component as suggested in https://github.com/dcos/dcos-ui/pull/94#discussion_r61862251